eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -65,6 +71,12 @@ jobs:
       - DisplayName: "Test on MacOS"
         OSVmImage: "macOS-10.15"
         TestTargetFramework: netcoreapp2.1
+      - ${{ if eq(parameters.TestCanary, 'true') }}:          
+        - DisplayName: "Test on Windows for .Net Framework - Canary region"
```

Comment:
Any reason you picked .NET Framework instead of .NET Core? I think .NET Core is our primary target so if we are only adding one additional test leg I would pick that.

Reply:
That was just my ignorance. 😀

Updated as you suggested.